### ISSUE TITLE
Typo in `Request::addModelAction()`

### ISSUE THREAD

**Issue by UlrichEckhardt** [itsgoingd/clockwork#669]
Heya!

I've been running PHPStan on the sources and it found an interesting bit of code:

```php
	public function addModelAction($model, $action, $data = [])
	{
		$this->modelActions[] = [
```

The field `modelActions` doesn't exist. However, a field `modelsActions` exists. This would be a simple fix, if it wasn't for the case that this method is totally unused, as far as I can tell. I'd also rather keep the name `modelActions` and rename all other cases. I'm not familiar enough to say how to progress here.

For the record, this was implemented as part of #397.

Cheers!

Uli

**Comment by itsgoingd** [comment]
Hey, these methods are part of the public api for people building their custom integrations, so they are not used in the Clockwork code itself. I'll fix the property name, thanks!

**Comment by itsgoingd** [comment]
Fixed in Clockwork 5.2.
